staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -808,15 +808,17 @@ func (g *genValidations) emitCallsToValidators(c *generator.Context, validations
 		nonfatalPtr := make([]validators.FunctionGen, 0, len(in))
 
 		for _, fg := range in {
-			flags := fg.Flags()
-			if flags&validators.IsFatal > 0 {
-				if flags&validators.PtrOK > 0 {
+			isFatal := (fg.Flags()&validators.IsFatal != 0)
+			isPtrOK := (fg.Flags()&validators.PtrOK != 0)
```

Comment:
Worth introducing a `isSet(validators.IsFatal)` receiver function for `FunctionFlags` ?

Reply:
good point, will add as a followup commit